addons/binding/org.openhab.binding.foobot/ESH-INF/thing/thing-types.xml

```diff
@@ -0,0 +1,90 @@
+
+        <channels>
+            <channel id="temperature" typeId="temperature" />
+            <channel id="humidity" typeId="humidity" />
+            <channel id="voc" typeId="voc" />
+            <channel id="co2" typeId="co2" />
+            <channel id="pm" typeId="pm" />
+            <channel id="gpi" typeId="gpi" />
+
+        </channels>
+
+        <config-description>
+            <parameter name="apikey" type="text" required="true">
+                <context>password</context>
+                <label>API Key</label>
+                <description>You can request your API Key from https://api.foobot.io/apidoc/index.html </description>
+            </parameter>
+
+            <parameter name="username" type="text" required="true">
+                <label>Username</label>
+                <description>The Username that you use to login to the Foobot account.</description>
+            </parameter>
+
+            <parameter name="mac" type="text" required="true">
+                <label>MAC Address</label>
+                <description>MAC address of your device. You can find it in the Foobot App under settings.</description>
+            </parameter>
+
+            <parameter name="refresh" type="integer" min="5" required="false">
+                <label>Refresh interval</label>
+                <description>Specifies the refresh interval in minutes.</description>
+                <default>30</default>
+            </parameter>
+
+        </config-description>
+    </thing-type>
+
+    <channel-type id="temperature">
+        <item-type>Number:Temperature</item-type>
+        <label>Temperature</label>
+        <description>Temperature</description>
+        <category>Temperature</category>
+        <state readOnly="true" pattern="%.1f °C"></state>
+    </channel-type>
+
+    <channel-type id="humidity">
+        <item-type>Number:Dimensionless</item-type>
+        <label>Humidity</label>
+        <description>Humidity Level</description>
+        <category>Humidity</category>
+        <state readOnly="true" pattern="%d %%"></state>
+    </channel-type>
+    <channel-type id="voc">
+        <item-type>Number</item-type>
+        <label>VOC</label>
+        <description>Volatile Organic Compounds Level</description>
+        <category>Sensor</category>
+        <state readOnly="true" pattern="%.1f ppb"></state>
+    </channel-type>
+    <channel-type id="co2">
+        <item-type>Number</item-type>
+        <label>CO2</label>
+        <description>Carbon dioxide Level</description>
+        <category>CarbonDioxide</category>
+        <state readOnly="true" pattern="%.1f ppm"></state>
+    </channel-type>
+    <channel-type id="pm">
+        <item-type>Number</item-type>
+        <label>PM</label>
+        <description>Particulate Matter Level</description>
+        <category>Sensor</category>
+        <state readOnly="true" pattern="%.1f μg/m&#179;"></state>
```

Comment:
We directly support the UTF-8 character here, see this line in a test case: https://github.com/eclipse/smarthome/blob/master/bundles/core/org.eclipse.smarthome.core.test/src/test/java/org/eclipse/smarthome/core/library/types/QuantityTypeTest.java#L244

Also, since this is a density the item-type should be `Number:Density`.